apps_download/models/product_product.py

```diff
@@ -0,0 +1,95 @@
+# -*- coding: utf-8 -*-
+
+import os
+import tempfile
+import shutil
+import logging
+import base64
+import subprocess
+from odoo.exceptions import ValidationError
+from odoo import models, fields, api
+_logger = logging.getLogger(__name__)
+
+
+class ProductProduct(models.Model):
+    _inherit = 'product.product'
+
+    dependent_product_ids = fields.Many2many('product.product', 'prto_validateoduct_dependent_rel', 'src_id', 'dest_id', string='Dependent Products')
```

Comment:
Max line length for OCA is 79 charachters, this is way to long.
Makes travis unhappy :)